### ISSUE TITLE
iOS10.3.3系统一直加载有图片的列表内存一直暴涨

### ISSUE THREAD

**Issue by Mamakou** [SDWebImage/SDWebImage#2695]
### New Issue Checklist

* [ ] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [ ] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [ ] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. ios / macos / tvos / watchos
 Platform Version        | e.g. 12.0 / 10.14.0 / 12.0 / 5.0
 SDWebImage Version      | e.g. 5.0.0 / 4.4.0
 Integration Method      | e.g. carthage / cocoapods / manually
 Xcode Version           | e.g. Xcode 10 / Xcode 9
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by bpoplauschi** [comment]
Can you translate the issue into English? if not, maybe @dreampiggy can take a look :)

**Comment by Mamakou** [comment]
thank you ! I I have solved the problem。Because the resolution of the picture is relatively large。
but,It is strange that other systems don't have this problem,such as iOS12.2, iOS11.0.
platform:iOS 10.3.3 ，SDWebImage Version：5.0.0，Integration Method：manually，Xcode Version：10.2,

UIImageView (WebCache)
- (void)sd_setImageWithURL:(nullable NSURL *)url
          placeholderImage:(nullable UIImage *)placeholder;
when load a large image, the Memory will surge.

**Comment by dreampiggy** [comment]
> It is strange that other systems don't have this problem
If you're using `UIImage imageNamed:` method, UIKit will using a **Lazy-Decoding-Rendering` policy, only when your UIView (Typically, the `CALayer`) is rendering in the display, it start to decoding the bitmap buffer and allocate the memory.

UIKit can not solve the memory usage from the stretch, because for everyone, bitmap RAM usage is just:

RAM = Bytes Per Pixel (4 for 32Bit-RGBA) * Pixel Count (Width * Height * Scale * Scale)

SDWebImage using a cache (<NSString, UIImage>), which keep all the bitmap buffer in the memory. It will be freed when the memory is under pressure. So It's OK and safe. Only if you found that the memory don't drop down when memory warning is trigged, we can say it's a bug.
